src/reference/config/bindings/binding-kafka.md

```diff
@@ -74,6 +74,7 @@ Conditional routes based on `kafka` `topic` names are used to route these networ
   - [sasl.mechanism\*](#sasl-mechanism)
   - [sasl.username](#sasl-username)
   - [sasl.password](#sasl-password)
+- [options.servers](#options-servers)
```

Comment:
Can you update the order of the ToC or the property section so they are in the same order. 
`### options.servers` is before `### options.sasl`, for instance.